src/flow.h

```diff
@@ -117,6 +117,9 @@ typedef struct AppLayerParserState_ AppLayerParserState;
 #define FLOW_TS_APP_UPDATED BIT_U32(29)
 #define FLOW_TC_APP_UPDATED BIT_U32(30)
 
+#define FLOW_TS_APP_UPDATE_NEXT BIT_U32(31)
+#define FLOW_TC_APP_UPDATE_NEXT BIT_U32(2)
```

Comment:
maybe remove the `vacancy` comment and say it is used now ;-)

Reply:
yeah